From f3331e2f34f019a9a39f3b1ed8a0cf3e1a649740 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
Date: Fri, 7 Feb 2025 00:34:30 +0100
Subject: [PATCH 24/27] riscv: dts: starfive: DeepComputing FML13V01 binman
 config

Add the DeepComputing Framework motherboard (FML13V01) to the binman
include for the starfive_visionfive2_defconfig.

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 arch/riscv/dts/starfive-visionfive2-binman.dtsi | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/arch/riscv/dts/starfive-visionfive2-binman.dtsi b/arch/riscv/dts/starfive-visionfive2-binman.dtsi
index 4cce001e80d..63289405fbb 100644
--- a/arch/riscv/dts/starfive-visionfive2-binman.dtsi
+++ b/arch/riscv/dts/starfive-visionfive2-binman.dtsi
@@ -16,6 +16,16 @@
 	itb {
 		fit {
 			images {
+				fdt-jh7110-deepcomputing-fml13v01 {
+					description = "jh7110-deepcomputing-fml13v01";
+					load = <0x40400000>;
+					compression = "none";
+
+					blob-ext {
+						filename = "dts/upstream/src/riscv/starfive/jh7110-deepcomputing-fml13v01.dtb";
+					};
+				};
+
 				fdt-jh7110-milkv-mars {
 					description = "jh7110-milkv-mars";
 					load = <0x40400000>;
@@ -58,6 +68,13 @@
 			};
 
 			configurations {
+				conf-jh7110-deepcomputing-fml13v01 {
+					description = "jh7110-deepcomputing-fml13v01";
+					firmware = "opensbi";
+					loadables = "uboot";
+					fdt = "fdt-jh7110-deepcomputing-fml13v01";
+				};
+
 				conf-jh7110-milkv-mars {
 					description = "jh7110-milkv-mars";
 					firmware = "opensbi";
-- 
2.43.0

